project/Settings.scala

```diff
@@ -0,0 +1,39 @@
+import sbt._
+import org.scalajs.sbtplugin.ScalaJSPlugin.autoImport._
+
+/**
+ * Application settings. Configure the build for your application here.
+ * You normally don't have to touch the actual build definition after this.
+ */
+object Settings {
+  /** The name of your application */
+  val name = "anduin-fsm"
+
+  /** The version of your application */
+  val version = "1.0"
+
+  /** Options for the scala compiler */
+  val scalacOptions = Seq(
+    "-Xlint",
+    "-unchecked",
+    "-deprecation",
+    "-feature"
+  )
+
+  /** Declare global dependency versions here to avoid mismatches in multi part dependencies */
+  object versions {
+    val scala = "2.12.2"
+    val log4js = "1.4.10"
+    val scalatest = "3.0.3"
```

Comment:
I often see people keep this as lower case (our stargazer uses this convention as well). I think it makes sense because most project name are in lower case and not upper case (e.g. `cats` or `scalajs`).